newsletter-october-2021.rst

```diff
@@ -0,0 +1,98 @@
+.. post:: October 6, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features in October.
+
+Read the Docs newsletter - October 2021
+=======================================
+
+Welcome to the latest edition of our monthly newsletter, where we
+share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- We have resumed sending our blog updates by email.
+  You can `subscribe to our newsletter <https://landing.mailerlite.com/webforms/landing/p8b7z2>`_
+  so you don't miss them.
+- Version 3 of our Embed API is now implemented,
+  thanks to :doc:`the CZI grant we received</czi-grant-announcement>`.
+  We will give more details about it soon.
+- We have reworked :doc:`our team page <readthedocs:team>`
+  to give you a better idea of who we are and how we work at the company.
+
+New features
+------------
+
+- You can now use our newer build specification
+  that uses a newer image based on Ubuntu 20.04
+  and gives more control over the versions of
+  Python, Node.js, Rust, and Go.
+  Projects like `pypa/cryptography <https://github.com/pyca/cryptography/pull/6330>`_
+  and `thebe <https://github.com/executablebooks/thebe/pull/472>`_
+  are already using it,
+  and we will publish documentation for it and make an announcement soon.
+- We have revamped our onboarding process.
+  If you are new to Read the Docs, you can now clone
```

Comment:
Is clone the right word? Seems overloaded with `git clone` -- perhaps `Fork`? GH says `Use this template`, so maybe use similar here?

Reply:
Woops, what I was thinking. Yes, Fork or Use are the right choices here.